src/SFA.DAS.ProviderApprenticeshipsService.Web/Validation/ApprenticeshipCoreValidator.cs

```diff
@@ -104,6 +103,7 @@ protected virtual void ValidateStartDate()
                 .Cascade(CascadeMode.StopOnFirstFailure)
                 .NotNull().WithMessage(_validationText.LearnStartDate01.Text).WithErrorCode(_validationText.LearnStartDate01.ErrorCode)
                 .Must(ValidateDateWithoutDay).WithMessage(_validationText.LearnStartDate01.Text).WithErrorCode(_validationText.LearnStartDate01.ErrorCode)
+                .Must(StartDateForTransferNotBeforeMay2018).WithMessage(_validationText.LearnStartDate06.Text).WithErrorCode(_validationText.LearnStartDate06.ErrorCode)
```

Comment:
The ValidationMessage LearnStartDate06 fits in with the naming pattern of all the other validation messages, but it's not very descriptive and it's not obvious from the name that e.g. LearnStartDate06 is the correct message for a transfer start date before May 2018! There seems no reason why we couldn't give the validation messages more descriptive names.